### ISSUE TITLE
Launcher and workflow names are not announced in the project dashboard

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#5256]
While each of the individual buttons on a launcher or workflow card (clone, show, launch, etc) are reachable by keyboard and have their own `title` attribute set, none of these titles mention the name of the launcher that the action happens to. If the user is not making sure to read all visible text, it would be very easy to lose your place and click the wrong button.. We should likely just include the item's name in these titles, so that instead of reading 'show launcher', it reads 'show launcher Launcher Name'
